plugins/comments.md

```diff
@@ -94,6 +115,8 @@ tinymce.init({
 **Result**: The commented text will be highlighted yellow.
 ![**Highlighted text**]({{site.baseurl}}/images/highlight.png)
 
+> Note: The values provided in the above example are the default values provided in Comments 2.0. These can be changed as desired in the `tinymce.init` script.
```

Comment:
As discussed, this isn't quite right. Comments v1. still changes annotations via content_style, but Comments v2 has different classes, and it's best to use content skins.

Reply:
Fixed